src/EditorFeatures/Core.Wpf/InlineRename/UI/SmartRename/SmartRenameUserInputComboBox.xaml.cs

```diff
@@ -109,6 +110,8 @@ private void GetSuggestionsButtonClick(object sender, RoutedEventArgs e)
             _smartRenameViewModel.IsSuggestionsPanelCollapsed = !_smartRenameViewModel.IsSuggestionsPanelCollapsed;
             if (_smartRenameViewModel.IsSuggestionsPanelExpanded)
             {
+                _smartRenameViewModel._cancellationTokenSource.Cancel();
+                _smartRenameViewModel._cancellationTokenSource = new CancellationTokenSource();
```

Comment:
Can you put this into `_smartRenameViewModel.GetSuggestionsCommand`?
(so that `CancellationTokenSource` could still be private)

Reply:
Make sense. Moved the reset to where the GetSuggestionsCommand is executed.